src/app/components/CommentsList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import axios from 'axios';
+import Comment from './Comment.js';
+
+export default class CommentsList extends React.Component {
+  constructor (props) {
+    super (props);
+    this.state = {
+      comments: []
+    }
+    axios.get('https://jsonplaceholder.typicode.com/comments')
+    .then (response => {
+      this.setState = ({comments: response.data})
```

Comment:
```suggestion
      this.setState({comments: response.data})
```
this.setState() - не свойство, а метод объекта